samples/star/src/main/kotlin/com/slack/circuit/star/overlay/BottomSheetOverlay.kt

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2022 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.slack.circuit.star.overlay
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.ModalBottomSheetLayout
+import androidx.compose.material.ModalBottomSheetValue
+import androidx.compose.material.rememberModalBottomSheetState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.slack.circuit.Overlay
+import com.slack.circuit.OverlayNavigator
+import kotlinx.coroutines.launch
+
+class BottomSheetOverlay<Model : Any, Result : Any>(
```

Comment:
Should we add a param for `Modifier` here? That way we could pass in the padding values from the parent Composable?

Reply:
I don't think so, we'd have to expose a Modifier on `Overlay.Content()`. It's sort of a similar case to `Ui.Content`, if the parent wants to impose a padding value they can just wrap it in a `Box` or similar that manages it for them. We don't really want the content here to know anything about the parent